sandstone/apps/filebrowser/handlers.py

```diff
@@ -259,11 +257,10 @@ def get(self):
                         # ignore the current file
                         continue
                     owner = getpwuid(stat_object.st_uid).pw_name
-                    file_size = str((float(stat_object.st_size) / 1024)) + " KiB"
                     curr_file['filename'] = i[0]
                     curr_file['filepath'] = i[1]
                     curr_file['owner'] = owner
-                    curr_file['size'] = file_size
+                    curr_file['size'] = self.fs.get_size(i[1])
```

Comment:
@saurabhsood91 Does this run for each file in a directory, each time a filetree dir is expanded?